libraries/rush-lib/assets/rush-init-subspace/.pnpmfile.cjs

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+/**
+ * When using the PNPM package manager, you can use pnpmfile.js to workaround
+ * dependencies that have mistakes in their package.json file.  (This feature is
+ * functionally similar to Yarn's "resolutions".)
+ *
+ * For details, see the PNPM documentation:
+ * https://pnpm.js.org/docs/en/hooks.html
+ *
+ * IMPORTANT: SINCE THIS FILE CONTAINS EXECUTABLE CODE, MODIFYING IT IS LIKELY TO INVALIDATE
+ * ANY CACHED DEPENDENCY ANALYSIS.  After any modification to pnpmfile.js, it's recommended to run
+ * "rush update --full" so that PNPM will recalculate all version selections.
+ */
+module.exports = {
+  hooks: {
+    readPackage
+  }
+};
+
+/**
+ * This hook is invoked during installation before a package's dependencies
+ * are selected.
+ * The `packageJson` parameter is the deserialized package.json
+ * contents for the package that is about to be installed.
+ * The `context` parameter provides a log() function.
+ * The return value is the updated object.
+ */
+function readPackage(packageJson, context) {
+  /*[LINE "HYPOTHETICAL"]*/ // The karma types have a missing dependency on typings from the log4js package.
+  /*[LINE "HYPOTHETICAL"]*/ if (packageJson.name === '@types/karma') {
+    /*[LINE "HYPOTHETICAL"]*/ context.log('Fixed up dependencies for @types/karma');
+    /*[LINE "HYPOTHETICAL"]*/ packageJson.dependencies['log4js'] = '0.6.38';
+    /*[LINE "HYPOTHETICAL"]*/
```

Comment:
You're using `copyFiles`, which won't handle these tokens. Look at https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/cli/actions/InitAction.ts